lib/uwb/protocols/fira/FiraDevice.cxx

```diff
@@ -106,10 +106,17 @@ UwbDeviceInformation::ToString() const
 {
     std::ostringstream ss;
     if (IsUwbStatusOk(Status)) {
+        ss << "Status: " << ::ToString(Status) << "\n";
```

Comment:
This will just print 'Ok' every time, I don't think we need it. In general, we've been excluding the status from structures once they've been decoded since the status isn't (in most cases) part of the state, but instead, describes whether obtaining the state was successful or not.

Reply:
I see, that seems perfectly reasonable to me. I'll remove this line